checkov/helm/runner.py

```diff
@@ -3,25 +3,70 @@
 import io
 import itertools
 import logging
-import operator
 import os
 import subprocess  # nosec
 import tempfile
-from functools import reduce
-from typing import Any
-
+from typing import Any, Type, Optional, List
 import yaml
 
+from checkov.common.graph.checks_infra.registry import BaseRegistry
+from checkov.common.graph.graph_manager import GraphManager
+from checkov.common.graph.db_connectors.networkx.networkx_db_connector import NetworkxConnector
+from checkov.common.graph.graph_builder.local_graph import LocalGraph
 from checkov.common.output.report import Report, CheckType
 from checkov.common.parallelizer.parallel_runner import parallel_runner
 from checkov.common.runners.base_runner import BaseRunner, filter_ignored_paths
 from checkov.helm.registry import registry
+from checkov.kubernetes.graph_builder.local_graph import KubernetesLocalGraph
 from checkov.kubernetes.runner import Runner as k8_runner
 from checkov.runner_filter import RunnerFilter
 
 K8_POSSIBLE_ENDINGS = [".yaml", ".yml", ".json"]
 
 
+class K8sHelmRunner(k8_runner):
+    def __init__(self, graph_class: Type[LocalGraph] = KubernetesLocalGraph,
+                 db_connector: NetworkxConnector = NetworkxConnector(),
+                 source: str = "Kubernetes",
+                 graph_manager: Optional[GraphManager] = None,
+                 external_registries: Optional[List[BaseRegistry]] = None) -> None:
+        super().__init__(graph_class, db_connector, source, graph_manager, external_registries)
+        self.chart_dir_and_meta = []
+        self.check_type = CheckType.HELM
+
+    def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=RunnerFilter(), collect_skip_comments=True, helmChart=None):
+        if external_checks_dir:
+            for directory in external_checks_dir:
+                registry.load_external_checks(directory)
```

Comment:
move this after `if not self.chart_dir_and_meta: ...`, no. need to do it, if there are no charts 😄